rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -452,10 +455,16 @@ if [ "$DKIM_ENABLE" = "true" ] && [ -f "$DKIM_PRIVATE_KEY" ]; then
   if [ -n "$SMTPD_MILTERS" ]; then SMTPD_MILTERS="${SMTPD_MILTERS},"; fi
   SMTPD_MILTERS="${SMTPD_MILTERS}unix:opendkim/opendkim.sock"
 fi
+
 if [ "$DMARC_ENABLE" = "true" ]; then
   if [ -n "$SMTPD_MILTERS" ]; then SMTPD_MILTERS="${SMTPD_MILTERS},"; fi
   SMTPD_MILTERS="${SMTPD_MILTERS}unix:opendmarc/opendmarc.sock"
 fi
+
+if [ -n "$RSPAMD_HOST" ]; then
+  SMTPD_MILTERS="inet:${RSPAMD_HOST}:${RSPAMD_PORT}"
+fi
```

Comment:
I think we should make Rspamd and OpenDKIM/OpenDMARC services mutually exclusive.